### PR TITLE
Adds modal window

### PR DESCRIPTION
**Pull request by Katended**
The following have been addressed in this pull request:

- Add a modal window for mobile and desktop screen sizes.
- Add close icon.

### CODE REVIEW

File: modal.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  list-style-type: disc;
+  left: 5%;
+  color: #7a869a;
+}
+
+@media (min-width: 768px) {
+  .popupContainer {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (AbelG101) [inline modal.css:124]: - [ ] Good job adding the popup but it seems like some of the text is getting clipped as shown below, so please make sure to add a scroll feature for a better user experience. ✔
- Your design:
5oI8FKYSjv
- Figma design:
  image

- [ ] I've also noticed the spacing in the elements is bit disproportionate and not as shown in the Figma design so please make sure you fix that as well I've tried to highlight some of the visible ones below:
  Inked5oI8FKYSjv

File: modal.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  background: #fff;
+  justify-content: start;
+  margin-right: 3%;
+  margin-left: 3%;
+  margin-top: 4%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (AbelG101) [inline modal.css:30]: - [ ] The mobile popup could also use some adjustment as I've noticed the current height you've specified is filling up almost the entire screen so please make sure the height is similar as specified in the Figma design🎨:
- Your design:
  image
- Figma design:
  image
- As you can see from the above Figma design the height doesn't take up the entire screen.

File: modal.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+.popupLinkContainer a:focus {
+  background-color: #020a58;
+}
+
+.popupLinkContainer a:hover {
+  background-color: #020a58;
+  color: white;
+  transition: 0.2s;
+}
```
Comment (AbelG101) [inline modal.css:72]: - [ ] I like that you're adding your own taste to the design here but you'll have to make sure you follow the given design at least at this stage as one of the objectives of this module is to give you the skill to implement any given design to near perfection 🎯
- Here's the Figma design for the hover state of the button:
  firefox_7KIb3eS8J1

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Katended 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- ## Please make sure you submit the appropriate PR for the milestone as I've noticed you submitted the [Mobile menu PR](https://github.com/Katended/mobilefirstproject2/pull/12) for the Popup milestone and it's causing confusion for us code reviewers when making the reviews.
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Katended  👋, 

Great work on making the changes requested previously 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
**- Please make sure you've addressed the previous changes that were requested before submitting for another review this will reduce your chances of getting requested for changes again. For now, you just have to make a few changes and you're on your way 🚀.**

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi  @Katended , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**